lib/spack/spack/compilers/cce.py

```diff
@@ -52,7 +52,7 @@ class Cce(Compiler):
 
     @classmethod
     def default_version(cls, comp):
-        return get_compiler_version(comp, '-V', r'[Vv]ersion.*(\d+(\.\d+)+)')
+        return get_compiler_version(comp, '-V', r'[Vv]ersion.*?(\d+(\.\d+)+)')
```

Comment:
Maybe I'm rusty with my regular expressions, but what is this question mark for? The preceding element (`.`) is already being matched 0 or more times. Isn't the question mark redundant?